pyvista/core/dataset.py

```diff
@@ -2133,6 +2143,19 @@ def find_closest_cell(self, point: Union[int, np.ndarray]) -> Union[int, np.ndar
         >>> indices.shape
         (5000,)
 
+        Find the point on the surface, including inside the cells,
+        that is closest to ``[0.1, 0.2, 0.3]``.  The closest point
+        on a perfect sphere, i.e. not discretized, with radius 0.5
+        is ``[0.134, 0.267, 0.401]``.
+
+        >>> index, closest_point = mesh.find_closest_cell(
+        ...     [0.1, 0.2, 0.3],
+        ...     return_closest_point=True
+        ... )
```

Comment:
To tie in with my earlier point about exactness: it's possible the closest point that we see here is actually a node in the mesh. I haven't checked. Not that this is necessarily an issue, but it might not highlight the "inside cells" aspect.

Reply:
I was trying to keep the docstring in one kind of theme, but I agree that a more to the point example is warranted since this behavior is complex to explain.

Reply:
This docstring example, now uses a 2D surface which is hopefully more clear what we are talking about.